docs/create-analystics-event.md

```diff
@@ -0,0 +1,143 @@
+# Create Analytics Event
+
+## Overview
+
+This document outlines the process for creating a new analytics event in MetaMask (Extension and Mobile)
+
+## Prerequisites
+
+- Before you begin, please refer to this [Working with Event Metrics](https://www.notion.so/consensys/Working-with-Event-Metrics-3b0b5a4308e64649a54a2864886c0412) and make sure you understand the context of the analytics events.
```

Comment:
Q: is this Notion page public for external contributors?

Reply:
it is not public but critical for the context, wdyt? should we move this out

Reply:
Can we copy it as MD in this repo?

Reply:
There are a lot of charts stuff. IMO external contributor won't add a metric event, we can remove this. wdyt?